Mgf/Mgf.cs

```diff
@@ -0,0 +1,134 @@
+﻿using MassSpectrometry;
```

Comment:
I suspect you'll need to edit the `mzLib.nuspec` file to get this to carry over into the nuget package

Reply:
I'm wondering if mzml and mgf really need their own projects or if we could just toss them in a folder in Mass Spectrometry...

Reply:
Done